packages/tools/src/enums/Events.ts

```diff
@@ -164,6 +164,10 @@ enum Events {
    */
   SEGMENTATION_DATA_MODIFIED = 'CORNERSTONE_TOOLS_SEGMENTATION_DATA_MODIFIED',
 
+  POLYSEG_CONVERSION_STARTED = 'CORNERSTONE_TOOLS_POLYSEG_CONVERSION_STARTED',
```

Comment:
Could you move to one type for polyseg, and just have a flag as to the progress (0 means start, and 100 means done, anything else is in between)?  Having too many types of events for a fairly fine grained change like that is harder to manage.